### DISCUSSION TITLE
未來的趨勢會是著重於邊緣運算的小型語言模型嗎？

### DISCUSSION THREAD

**Discussion by JaYanlloz** [1101-GS4719-C/Python-Community-GS4719-C#2178] [科技新知]
近幾年AI發展得很快，但大家都在關注大型語言模型(LLM)，但目前看來小型語言模型也有崛起的趨勢，這種模型適合邊緣運算，就是設備自己處理數據，不用依賴雲端。這樣的好處是反應速度快，也能減少對網路的依賴，像智慧家居、手機等都能用上。聯發科和高通都認為，小型語言模型對這些邊緣設備來說非常適合，因為它算力需求低，運行成本也比較便宜。

未來可能會有更多的設備使用這類小型模型，因為它們在日常應用場景下已經足夠應付，不需要太強大的雲端AI來處理每件事。對台灣來說，這也是一個不錯的發展方向，可以抓住機會發展邊緣運算和小型模型相關技術，未來市場潛力很大 。

**Comment by ShuliANN** [comment]
以數據隱私與安全的需求的角度來說，大家越來越關注個人數據的隱私問題，在邊緣設備上進行數據處理成為了一種重要解決方案。小型語言模型能夠在本地設備上處理敏感信息，減少數據傳輸至雲端的風險，從而更好地保護用戶隱私。這對於涉及個人資訊的應用（如健康監測、語音助手等）尤其關鍵。

**Comment by 112121py** [comment]
聽起來是一個蠻不錯的想法，也許也是可以跟大型語言模型的應用做串接，像是在日常需求下，我們就使用小型語言模型，如果有更多的需求，可以再去做請求，轉接到大型語言模型做使用，有點像客服中心的那種感覺，不過資安問題就蠻需要被好好的解決了，畢竟大家也不想讓自己的資訊暴露在危險之下
